From: Jeremy Allison <jra@samba.org>
Date: Thu, 13 Dec 2007 02:45:20 +0000 (-0800)
Subject: Fix bug #4784. Patch from Steve Langasek <vorlon@debian.org>.
X-Git-Url: http://gitweb.samba.org/?p=samba.git;a=commitdiff_plain;h=244230eb619c6699e962bb24f8f6e4dd2a1235c7

Fix bug #4784. Patch from Steve Langasek <vorlon@debian.org>.
Jeremy.
---

diff --git a/source/client/umount.cifs.c b/source/client/umount.cifs.c
index 47ddd1e..70829eb 100644
--- a/source/client/umount.cifs.c
+++ b/source/client/umount.cifs.c
@@ -131,7 +131,7 @@ static int umount_check_perm(char * dir)
 		printf("user unmounting via %s is an optional feature of",thisprogram);
 		printf(" the cifs filesystem driver (cifs.ko)");
 		printf("\n\tand requires cifs.ko version 1.32 or later\n");
-	} else if (rc > 0)
+	} else if (rc != 0)
 		printf("user unmount of %s failed with %d %s\n",dir,errno,strerror(errno));
 	close(fileid);
 

